arrays/arrays.c

```diff
@@ -20,12 +20,14 @@ typedef struct Array {
  * Allocate memory for a new array
  *****/
 Array *create_array (int capacity) {
-  // Allocate memory for the Array struct
-
-  // Set initial values for capacity and count
-
-  // Allocate memory for elements
-
+    // Allocate memory for the Array struct
+    Array *arr = malloc(sizeof(Array));
+    // Set initial values for capacity and count
+    arr->capacity = capacity;
+    arr->count = 0;
+    // Allocate memory for elements
+    arr->elements = malloc(sizeof(char) * capacity);
```

Comment:
`sizeof(char *)` :) you're storing pointers to chars `(char *)`, not chars themselves